src/platform/forms-system/src/js/widgets/DateWidget.jsx

```diff
@@ -83,7 +84,7 @@ export default class DateWidget extends React.Component {
     const { month, day, year } = this.state.value;
     let daysForSelectedMonth;
 
-    const monthYear = options.monthYear;
```

Comment:
the linter is preventing me from removing this change.  It is a good change, but it is also unrelated to my commit.  Not sure what to do but ignore it.